source/includes/_opportunities.md

```diff
@@ -295,7 +295,7 @@ You can GET a list of all created Opportunities from [Opportunities/all](#get-al
 
 `GET https://api.apollo.io/v1/opportunities/REPLACE_WITH_OPPORTUNITY_ID`
 
-## Update Opportunity
+## Update opportunity
```

Comment:
Line no 294 
`You can GET a list of all created Opportunities from Opportunities/all` -> `You can GET a list of all created opportunities from opportunities/all`

Reply:
Line no 365
Same as above